docs/deploy-local.md

```diff
@@ -100,10 +101,11 @@ In addition to the Kubernetes version requirement in [Deploying CF for K8s](depl
    - Include the [remove-resource-requirements.yml](../config-optional/remove-resource-requirements.yml) and
      [remove-ingressgateway-service.yml](../config-optional/remove-ingressgateway-service.yml)
      overlay files in the set of templates to be deployed. This can be achieved by
-     using the following command instead of running the install-cf.sh script:
+     using the following commands:
 
      ```bash
-     kapp deploy -a cf -f <(ytt -f config -f <cf_install_values_path> -f config-optional/remove-resource-requirements.yml -f config-optional/remove-ingressgateway-service.yml)
+     $ ytt -f config -f config-optional/remove-resource-requirements.yml -f config-optional/remove-ingressgateway-service.yml -f <cf_install_values_path> > /tmp/cf-for-k8s-rendered.yml
```

Comment:
In a very split team vote, our PM made the executive decision for us to default to no `$ ` prompts in documentation, unless we're explicitly showing both commands and output lines in the same section (like `minikube ip` block above)